chains/sophonsepolia/metadata.yaml

```
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api-explorer-verify.testnet.sophon.xyz/contract_verification
+    family: zksync
```

`zksync` explorer type is not valid schema yet
```suggestion
    family: etherscan
```